bundle-size/api.js

```diff
@@ -127,12 +132,20 @@ function extraBundleSizesSummary(
   if (bundleSizeDeltasRequireApproval.length) {
     output +=
       '\n## Bundle size changes that require approval\n' +
-      bundleSizeDeltasRequireApproval.join('\n');
+      // If files require approval, we're firstly interested in the largest
+      // delta, so we sort desc
+      sortBundleSizeItems(bundleSizeDeltasRequireApproval, 'desc')
+        .map(formatBundleSizeItem)
+        .join('\n');
   }
   if (bundleSizeDeltasAutoApproved.length) {
     output +=
       '\n## Auto-approved bundle size changes\n' +
-      bundleSizeDeltasAutoApproved.join('\n');
+      // If files don't require approval, we're firstly interested in the
+      // smallest delta, so we sort asc
+      sortBundleSizeItems(bundleSizeDeltasAutoApproved, 'asc')
+        .map(formatBundleSizeItem)
+        .join('\n');
```

Comment:
Eh, I think it's fine to sort both cases as desc - it creates a consistent reading experience 🤷‍♂️